test/Microsoft.AspNetCore.Mvc.FunctionalTests/ViewEngineTests.cs

```diff
@@ -247,7 +247,7 @@ public static IEnumerable<object[]> RazorViewEngine_RendersPartialViewsData
         }
 
         [Fact]
-        public async Task RazorViewEngine_RendersViewsFromEmbeddedFileProvider()
+        public async Task RazorViewEngine_RendersViewsFromEmbeddedFileProvider_WhenLookedupByName()
```

Comment:
Suggest a `[Theory]` would make consistency with `RazorViewEngine_RendersViewsFromEmbeddedFileProvider_WhenLookedupByPath()` clear.

Reply:
Oops. This looks better but shouldn't be `async`. Remove that and all will be well w/ the compiler.